### PR TITLE
Wait 1 second between attempts to check workers to not hammer the CPU

### PR DESCRIPTION
**Pull request by nathanielks**
I'm seeing the Runner process hitting high CPU percentages (40-50%) whenever it's been instructed to stop. Adding this 1 second wait to not hit the CPU so hard.

### CODE REVIEW

**Comment by rmccue** [comment]
FWIW, we could probably wait less than a full second; that sleep is mainly to avoid hammering the DB server, but the CPU should be able to take a bit more (maybe not every cycle tho)

**Comment by nathanielks** [comment]
@rmccue what's your preference? 1/10th, 1/4, 1/2 of a second?

**Comment by rmccue** [comment]
As fast as we can without destroying the CPU; you want the process to exit as quickly as possible.

**Comment by nathanielks** [comment]
@rmccue Updated to 1/10th of a second 👍 I'm picking out of thin air, so I reckon that's reasonably fast as well as a reasonable break.

File: inc/class-runner.php [hunk 1]
```diff
@@ -160,6 +160,7 @@ public function terminate( $signal ) {
 		// Wait and clean up
 		while ( ! empty( $this->workers ) ) {
 			$this->check_workers();
+			usleep(100000);
```
Comment (rmccue) [inline inc/class-runner.php]:
```suggestion
			usleep( 100000 );
```

**Comment by nathanielks** [comment]
Doneskies 👍

**Pull request merged**